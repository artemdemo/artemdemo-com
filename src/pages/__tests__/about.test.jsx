import React from 'react';
import renderer from 'react-test-renderer';
import About from '../about';

jest.mock('../../components/Layout/Layout');

describe('About', () => {
    it('should render', () => {
        const tree = renderer.create(
            <About
                data={{
                    site: {
                        siteMetadata: {
                            title: 'Some title',
                            description: 'Some description'
                        },
                    },
                }}
            />
        ).toJSON();

        expect(tree).toMatchSnapshot();
    });
});