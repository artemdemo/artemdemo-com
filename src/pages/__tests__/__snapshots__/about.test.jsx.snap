// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About should render 1`] = `
<div
  data-description="Some description"
  data-metaTitle="Some title"
  data-mock="Layout"
  data-title="Some title"
>
  <p>
    Hi guys,
  </p>
  <p>
    My name is Artem and I'm Frontend developer. I leave and work in Tel Aviv, while truing to build useful things. I decided to start this blog for number of reasons:
  </p>
  <p>
    First, I really believe in knowledge sharing. This is what continue to improve the web industry and each developer can only benefit from that (the one who shares and the one who can find useful information on the internet).
  </p>
  <p>
    Second, is better understanding of concepts. You truly start to understand something, only when you start to teach it. I know, that for some it could look not intuitive, but it's true. By explaining complicated concepts we start to understand them better by ourselves.
  </p>
  <p>
    If you're interested in my open source projects, or code examples, you can checkoutÂ 
    <a
      href="https://github.com/artemdemo"
      target="_blank"
    >
      my github account
    </a>
  </p>
</div>
`;
